keymaps/python-tools.cson

```diff
@@ -7,7 +7,7 @@
 
 # For more detailed documentation see
 # https://atom.io/docs/latest/behind-atom-keymaps-in-depth
-'atom-workspace':
+'atom-text-editor[data-grammar~="python"]:not([mini])':
```

Comment:
To not to use the keybinding on one-line text inputs.

ref: https://discuss.atom.io/t/what-is-the-purpose-of-atom-text-editor-not-mini-in-keymap-cson/18273